httputil/client.go

```diff
@@ -30,3 +39,52 @@ func GetRequestIP(req *http.Request) string {
 
 	return strings.ReplaceAll(addresses[0], " ", "")
 }
+
+func DefaultTransport() *http.Transport {
```

Comment:
I think we're missing a few settings that are set in the `http.DefaultTransport` instead, `ExpectContinueTimeout` to make an example among others. Can you please review them and add the missing ones?